tests/l2/README.md

```diff
@@ -73,8 +73,10 @@ This workload runs [qatlib](https://github.com/intel/qatlib) sample tests. Refer
 
 *	Deploy and execute the workload.
 
-    * Run with IPC_LOCK capability for pod
+    * Run with IPC_LOCK capability for pod using custom SCC and rbac with service account
```

Comment:
Create SCC intel-qat-scc for Intel QAT technology based workload if this SCC is not created

Reply:
Create SCC intel-qat-scc for Intel QAT based workload, if this SCC is not created